README.md

```diff
@@ -114,7 +122,8 @@ Your query here
 5. How many offices are in the state of Colorado (United States).
 
 ```
-Your query here
+select count(*) list_of_office from offices o 
+where state_id = (select id from states s where name ilike 'colorado');
```

Comment:
I think the only case when an ILIKE command should be used is when there's a pattern matching task to be done. Here I don't understand why a simple `name = 'Colorado'` wouldn't be enough.